data/org.freedesktop.portal.InputCapture.xml

```diff
@@ -114,6 +116,101 @@
+          more information about the session handle.
+
+        Supported keys in the @results vardict may include:
+
+        * ``session_handle`` (``o``)
+
+          The session handle. An object path for the
+          :ref:`org.freedesktop.portal.Session` object representing the created
+          session. Required.
+
+        This method was added in version 2 of this interface.
+    -->
+    <method name="CreateSession2">
+      <annotation name="org.qtproject.QtDBus.QtTypeName.In0" value="QVariantMap"/>
+      <arg type="a{sv}" name="options" direction="in"/>
+      <arg type="a{sv}" name="results" direction="out"/>
+    </method>
+
+    <!--
+        Start:
+        @session_handle: Object path for the :ref:`org.freedesktop.portal.Session` object
+        @parent_window: Identifier for the application window, see :doc:`window-identifiers`
+        @options: Vardict with optional further information
+        @handle: Object path for the :ref:org.freedesktop.portal.Request object representing this call
+
+        Start the input capture session. This will typically result in the portal
+        presenting a dialog letting the user decide whether they want to allow the
+        input of the session to be captured, and what capabilities to support.
+
+        This method may only be called once on a session previously created with CreateSession2.
+        Sessions created by the deprecated CreateSession are immediately started.
+
+        Supported keys in the @options vardict include:
+
+        * ``handle_token`` (``s``)
+
+          A string that will be used as the last element of the @handle. Must be a valid
+          object path element. See the :ref:`org.freedesktop.portal.Request` documentation for
+          more information about the @handle.
+
+        * ``capabilities`` (``u``)
+
+          Bitmask of requested capabilities, see the SupportedCapabilities property.
+          This value is required and must not be zero.
+
+        The following results get returned via the :ref:`org.freedesktop.portal.Request::Response` signal:
+
+        * ``capabilities`` (``u``)
+
+          The capabilities available to this session. This is always a
+          subset of the requested capabilities.
+          See the SupportedCapabilities property for details. Note that
+          while a capability may be available to a session, there is no
+          guarantee a device with that capability is currently available
+          or if one does become available that it will trigger input capture.
+
+          It is best to view this set as a negative confirmation - a
+          capability that was requested but is missing is an indication that
+          this application may not capture events of that capability.
+
+        * ``clipboard_enabled`` (``b``)
+
+          A boolean for whether the clipboard was enabled ('true') or not('false').
+          See the :ref:`org.freedesktop.portal.Clipboard` documentation for more
+          information.
```

Comment:
We need to touch up clipboard docs  to mention input capture and/or this otherwise you are stuck in a referencing loop as clipboard currently reads:


> This portal does NOT create it’s own session. Instead, it offers existing sessions created from other portals the option to integrate with the clipboard. For whether this interface is supported for a given session, refer to that portal’s documentation. See Remote Desktop  to integrate clipboard with the remote desktop session.